uuid/mod.ts

```diff
@@ -2,23 +2,87 @@
 // This module is browser compatible.
 
 /**
- * Generators and validators for UUIDs for versions v1, v3, v4 and v5.
+ * Generators and validators for
+ * {@link https://www.rfc-editor.org/rfc/rfc4122.html | RFC4122} UUIDs for
+ * versions v1, v3, v4 and v5.
  *
- * Use {@linkcode crypto.randomUUID} for v4 generating v4 UUIDs.
+ * Use the built-in
+ * {@linkcode https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID | crypto.randomUUID()}
+ * function instead of this package, if you only need to generate v4 UUIDs.
  *
- * Based on {@linkcode https://www.npmjs.com/package/uuid | npm:uuid}, which is
- * based on {@link https://www.rfc-editor.org/rfc/rfc4122.html | RFC 4122}.
+ * Based on {@linkcode https://www.npmjs.com/package/uuid | npm:uuid}.
  *
- * Support for RFC4122 version 1, 3, 4, and 5 UUIDs
+ * ```ts
+ * import { v5, NAMESPACE_DNS, NIL_UUID } from "@std/uuid";
+ * import { assert, assertFalse } from "@std/assert";
  *
- * This module is browser compatible.
+ * const data = new TextEncoder().encode("deno.land");
+ * const uuid = await v5.generate(NAMESPACE_DNS, data);
+ *
+ * assert(v5.validate(uuid));
+ * assertFalse(v5.validate(NIL_UUID));
+ * ```
  *
  * @module
  */
 
 export * from "./common.ts";
 export * from "./constants.ts";
+/**
+ * Generator and validator for
+ * {@link https://www.rfc-editor.org/rfc/rfc9562.html#section-5.1 | UUIDv1}.
+ *
+ * @example Usage
+ * ```ts
+ * import { v1 } from "@std/uuid";
+ * import { assert } from "@std/assert/assert";
+ *
+ * const uuid = v1.generate();
+ * assert(v1.validate(uuid as string));
+ * ```
+ */
 export * as v1 from "./v1.ts";
+/**
+ * Generator and validator for
+ * {@link https://www.rfc-editor.org/rfc/rfc9562.html#section-5.3 | UUIDv3}.
+ *
+ * @example Usage
+ * ```ts
+ * import { v3, NAMESPACE_DNS } from "@std/uuid";
+ * import { assert } from "@std/assert/assert";
+ *
+ * const data = new TextEncoder().encode("deno.land");
+ * const uuid = await v3.generate(NAMESPACE_DNS, data);
+ * assert(v3.validate(uuid));
+ * ```
+ */
 export * as v3 from "./v3.ts";
+/**
+ * Validator for
+ * {@link https://www.rfc-editor.org/rfc/rfc9562.html#section-5.4 | UUIDv4}.
+ *
+ * @example Usage
+ * ```ts
+ * import { v4 } from "@std/uuid";
+ * import { assert } from "@std/assert/assert";
+ *
+ * const uuid = crypto.randomUUID();
+ * assert(v4.validate(uuid));
+ * ```
+ */
 export * as v4 from "./v4.ts";
+/**
+ * Generator and validator for
+ * {@link https://www.rfc-editor.org/rfc/rfc9562.html#section-5.5 | UUIDv5}.
+ *
+ * @example Usage
+ * ```ts
+ * import { v5, NAMESPACE_DNS } from "@std/uuid";
+ * import { assert } from "@std/assert/assert";
+ *
+ * const data = new TextEncoder().encode("deno.land");
+ * const uuid = await v5.generate(NAMESPACE_DNS, data);
+ * assert(v5.validate(uuid));
+ * ```
+ */
 export * as v5 from "./v5.ts";
```

Comment:
These changes actually don't register as docs for `v1`, etc., and I'm not sure how to make it do so. This seems like a bug. Any ideas @crowlKats or @dsherret?

Reply:
How about the below workaround?

```ts
import * as v1_ from "./v1.ts";
/**
 * ...Docs...
 */
export const v1 = v1_;
```

Reply:
Looks like the below is even better?:

```ts
import { generate as generateV1, validate as validateV1 } from "./v1.ts";
/**
 * ...docs
 */
export const v1 = {
  generate: generateV1,
  validate: validateV1,
};
```

Reply:
Awesome. Thanks!